tests/publiccloud/check_services.pm

```diff
@@ -24,6 +24,12 @@ sub run {
     # Debug
     $instance->ssh_script_run('systemctl --no-pager list-units');
 
+    # Check if there are any failed services
+    my $failed_services = $instance->ssh_script_output('systemctl --failed');
+    unless ($failed_services =~ /0 loaded units listed/) {
+        record_info("Failing services!", $failed_services);
+        die("There are some failed services!");
```

Comment:
No need to add record_info, using `die`:
```suggestion
        die("There are some failed services! See:\n" . $failed_services);
```

Reply:
It was like this before and I was asked to change it and add a `record_info`...